mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -63,6 +65,7 @@
     private final Field[] idFields;
     private final Path[] idPaths;
     private final String[] mongoIdFields;
+    private final String txid=ObjectId.get().toString();
```

Comment:
Some places use txid, some use docver (docversion,etc). I think maybe all of them should just be called docVer/docVersion to be clear they are the same thing.